utils/backport.py

```diff
@@ -16,23 +16,32 @@
 parser.add_argument("pr", type=int, help="the # of the PR to backport")
 parser.add_argument("version", help="the release version to target with the backport")
 parser.add_argument("remote", default="origin", help="the git remote to use (defaults to origin)")
+parser.add_argument("--skip", default=[], action="append", help="don't cherry-pick these commits")
 args = parser.parse_args()
 
 title = json.loads(
     subprocess.check_output(["gh", "pr", "view", str(args.pr), "--json", "title"], text=True)
 )["title"]
 commits = json.loads(
     subprocess.check_output(
-        ["gh", "api", f"repos/{{owner}}/{{repo}}/pulls/{args.pr}/commits"], text=True
+        ["gh", "api", "--paginate", f"repos/{{owner}}/{{repo}}/pulls/{args.pr}/commits"], text=True
     )
 )
-print(f'Backporting {len(commits)} commits from "{title}"')
+
+# NB. It's tempting to do something like `set(commit_hashes) -= set(args.skip)`
+# here, but we must preserve the order of commits, and it's not worth pulling
+# in an ordered-set implementation just for this.
+commit_hashes = [commit["sha"] for commit in commits]
```

Comment:
if you like list comprehensions... :)

```suggestion
commit_hashes = [commit["sha"] for commit in commits if commit["sha"] not in args.skip]
```

Reply:
Delightful.  Thanks for the reminder.